### PR TITLE
Add docker image signing - INT-7480

### PR DESCRIPTION
**Pull request by eduard-tita**
Jira: https://issues.sonatype.org/browse/INT-7480
Build: https://jenkins.ci.sonatype.dev/job/insight/job/insight-brain/job/docker/job/docker-nexus-iq-server-feature/job/INT-7480_sign_docker_image_for_iq_server/

### CODE REVIEW

**Review by amoreno-sonatype — APPROVED** [review]
+1 LGTM

**Review by hectorhuol — APPROVED** [review]
+1

File: Jenkinsfile [hunk 1]
```diff
@@ -163,34 +163,44 @@ node('ubuntu-zion-legacy') {
...
-          OsTools.runSafe(this, "docker tag ${imageId} ${organization}/${dockerHubRepository}:latest")
-          OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:${version}-slim")
-          OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:latest-slim")
-          OsTools.runSafe(this, """
+        OsTools.runSafe(this, "mkdir -p '${env.WORKSPACE_TMP}/.dockerConfig'")
+        OsTools.runSafe(this, "cp -n '${env.HOME}/.docker/config.json' '${env.WORKSPACE_TMP}/.dockerConfig' || true")
+        withEnv(["DOCKER_CONFIG=${env.WORKSPACE_TMP}/.dockerConfig", 'DOCKER_CONTENT_TRUST=1']) {
+          withCredentials([
+              string(credentialsId: 'nexus-iq-server_dct_reg_pw', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE'),
+              file(credentialsId: 'nexus-iq-server_dct_gun_key', variable: 'REGISTRY_KEY'),
```
Comment (tneer) [inline Jenkinsfile]: Minor, but maybe REPOSITORY_KEY is a slightly more meaningful variable name.

**Review by tneer — APPROVED** [review]

**Review by jflinchbaugh — APPROVED** [review]
+1

**Pull request merged**